From 096877aa28d4427148a200134b03cbc8869943d7 Mon Sep 17 00:00:00 2001
From: Joshua Watt <JPEWhacker@gmail.com>
Date: Wed, 14 Mar 2018 16:20:04 -0500
Subject: [PATCH] checkbashisms.pl: Invoke perl from PATH

Search PATH for the perl interpreter when invoking the script

Signed-off-by: Joshua Watt <JPEWhacker@gmail.com>
Upstream-Status: inappropriate
---
 scripts/checkbashisms.pl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/checkbashisms.pl b/scripts/checkbashisms.pl
index 76578f86..2b97aae4 100755
--- a/scripts/checkbashisms.pl
+++ b/scripts/checkbashisms.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 # This script is essentially copied from /usr/share/lintian/checks/scripts,
 # which is:
-- 
2.14.3

